andela/travel_tool_front
# #162438580 Create Docker Environment for end-to-end Tests

#### What does this PR do?
- Creates docker environment for running end-to-end tests
- Adds make command for running end-to-end tests using docker
- Changes docker images for running the application to node:10.14.1

#### Description of Task to be completed?
Create docker container which would be running both the frontend and the backend in order for end-to-end tests to get executed.

#### How should this be manually tested?
1. Pull this branch and chekout
2. Run `make help` and check how to run end-to-end tests
3. Run end-to-end test using the make command

#### Any background context you want to provide?
1. It might take a bit long to run end-to-end tests for the first time.
2. This PR depends on https://github.com/andela/travel_tool_back/pull/209.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162438579
https://www.pivotaltracker.com/story/show/162438580

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
